google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java

```diff
@@ -47,4 +47,11 @@
       "cloud.debugger.close.continue.listen";
   public static final String CLOUD_DEBUGGER_NOTIFY_BREAKPOINT_LIST_CHANGE =
       "cloud.debugger.notify.breakpoint.list.change";
+
+  public static final String APP_ENGINE_OLD_PLUGIN_NOTIFICATION =
+      "appengine.oldplugin.notification";
+  public static final String APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK =
```

Comment:
Well, that's actually tracked by `appengine.oldplugin.deactivated` below. This event is used to track a click on the link in the notification that's shown to the user (which just opens a dialog with more detail, and the button that actually disables the plugin). See my above comment for more detail.